test/common/validity_tests.rb

```diff
@@ -0,0 +1,217 @@
+# frozen_string_literal: true
+
+module RGeo
+  module Tests
+    module Common
+      module ValidityTests
+        def test_invalid_point_invalid_coordinate
+          assert(@invalid_point.invalid_reason.include?(RGeo::ImplHelper::TopologyErrors::INVALID_COORDINATE))
```

Comment:
I'm a bit unsure why you do not instantiate `@invalid_point` in the module? same for most instance variables there!

To do so, you could use memoization for instance

```ruby
def invalid_point
  return @invalid_point if defined?(@invalid_point)

  @invalid_point = ...
end

Reply:
Good point. Done!